roles/patch_host_config/templates/patch-worker-partition.j2

```diff
@@ -0,0 +1,28 @@
+#jinja2:trim_blocks: True, lstrip_blocks: True
+{
+  "ignition": {
+    "version": "3.2.0" 
+  },
+  "storage": {
+    "disks": [
+      {% for disk in disks %}
+      {
+        "device": "{{ disk.device }}",
+        "partitions": [ 
+          {% set partitions = ((disk.extra_partitions | default({})).items() | list) %}
+          {% if disk.device == (installation_disk_path | default(host.json.installation_disk_path)) %}
+          {{ (disks_rhcos_root | tojson) + (((partitions | length) > 0) | ternary(',','')) }}
+          {% endif %}
+          {% for label, size in partitions %}
+          {
+            "label": "{{ label | string }}",
+            "sizeMiB": {{ size | int }}
+          }{{ ((loop.index == (partitions | length)) | ternary('',',')) }}
+          {% endfor %}
+        ],
+        "wipeTable": false
+      }{{ ((loop.index == (disks | length)) | ternary('',',')) }}
```

Comment:
Should there be a new line between the curlies `}{{`?

Reply:
No we don't want a new line there as the comma that creates needs to be on end of the line loop is creating.

Reply:
👍